### DISCUSSION TITLE
BackGroundMusic volume slider and more button sounds please

### DISCUSSION THREAD

**Discussion by TooGeekCreations** [OnionUI/Onion#1106] [Ideas]
-Any chance to add a slider in the settings menu or an On-Off button to turn off the theme background music without turning off the change.wav sound?

-Can we have more button sounds in the MenuUI not just the change.wav so with different button press different sound effect plays (like: d_pad.wav select.wav, start.wav, a.wav, b.wav, x.wav, y.wav menu.wav)

**Comment by Aemiii91** [comment]
Not really possible since the frontend is closed source. We might be able to add a bgm toggle to the theme switcher.

**Reply by TooGeekCreations** [reply]
"We might be able to add a bgm toggle to the theme switcher."
That would be good as well.
Thanks for the answer.
